components/Comments/Comments.vue

```diff
@@ -57,12 +62,12 @@
         }
       },
       onReply (comment) {
-        this.replyComment = comment
+        this.replyComment = comment;
```

Comment:
Please remove all semicolons as they are not js standard style conform.

Reply:
done